SoftLayer/order_utils.py

```diff
@@ -0,0 +1,145 @@
+"""Utility functions for mapping packages and categories to complex types."""
+
+
+def get_package_type_map():
```

Comment:
would it make sense to add the virtual server and bare metal server types here?

Reply:
Apparently what I've been told, we can't map directly from that package to that order type